src/Controller/ProfileController.php

```diff
@@ -67,4 +68,21 @@ public function edit(
             'formpassword' => $formpassword
         ]);
     }
+
+    /**
+     * @Route("/profile/delete", name="profile_delete")
+     */
```

Comment:
This controller should probably not be accessible unless you are properly connected to the site.